<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>819</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>819</width>
    <height>510</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>RackStore Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget1" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>809</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
	background:#FFFFFF;
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>309</width>
       <height>500</height>
      </rect>
     </property>
     <property name="text">
      <string>image</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>130</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>200</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>216</y>
       <width>32</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>280</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
     </layout>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>538</x>
       <y>430</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>icon</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>460</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
      </font>
     </property>
     <property name="text">
      <string>Need help ?</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>0</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
